docs/openapi.yaml

```diff
@@ -501,11 +501,76 @@ paths:
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Series'
+                $ref: '#/components/schemas/SerialPublication'
         400:
           $ref: '#/components/responses/400'
         502:
           $ref: '#/components/responses/502'
+  /serial-publication/{identifier}/{year}:
+    get:
+      x-amazon-apigateway-integration:
+        uri:
+          Fn::Sub: arn:aws:apigateway:${AWS::Region}:lambda:path/2015-03-31/functions/${FetchSerialPublicationByIdentifierAndYearFunction.Arn}/invocations
+        httpMethod: POST
+        requestParameters:
+          integration.request.path.identifier:
+            'method.request.path.identifier'
+          integration.request.path.year:
+            'method.request.path.year'
+        cacheKeyParameters:
+          - 'method.request.path.identifier'
+          - 'method.request.path.year'
+        type: "AWS_PROXY"
+      tags:
+        - SerialPublication
+      summary: Fetch journal or series by identifier and year
+      description: Returns a single Journal or Series with information for a specific year
+      operationId: FetchSerialPublicationByIdentifierAndYear
+      parameters:
+        - name: identifier
+          in: path
+          description: identifier of journal/series to return
+          required: true
+          schema:
+            type: string
+          example: '151f411d-68cd-4c7a-9cbb-daf00e0326ce'
+        - name: year
+          in: path
+          description: year for journal/series data
+          required: true
+          schema:
+            type: integer
+            description: can maximum be set to next year
+          example: '1910'
+      responses:
+        200:
+          description: successful operation
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/SerialPublication'
+              examples:
+                objectExample:
+                  $ref: '#/components/examples/JournalResponseExample'
```

Comment:
Possible to link multiple examples, `SeriesExample` in addition to `JournalResponseExample`.

Reply:
Done!